src/platform/forms-system/src/js/review/ObjectField.jsx

```diff
@@ -140,11 +140,13 @@ class ObjectField extends React.Component {
       },
     );
 
-    let title = formContext?.pageTitle;
-    if (!formContext?.hideTitle && typeof title === 'function') {
```

Comment:
I'm wondering if some pages still want to respect the hideTitle? this affects basically every form and some may have different formatting with their content. it would be good to test this on several different forms outside of simple-forms to make sure this is what we want

Reply:
I just fixed the title-prop (L175) for the custom <ObjectReviewField>.  L187 already has the hideTitle check, which is why I could remove that from L144.

Also, the unit-test all passed, incl. 'should hide title' (L212 in the spec).